Enable token exchange (#84)
eclipse/che-operator: README.md

This applies both to Run and Debug.

### Pre-Reqs: /tmp/keycloak_provision and oauth_provision files

> The operator grabs these files and replaces values to get a string used as exec command to configure Keycloak.

Reviewer: ```suggestion
The operator takes these files and replaces values to get a string used as the `exec` command to configure Keycloak.
```

Make sure you run the following before running/debugging:

```
cp deploy/keycloak_provision /tmp/keycloak_provision
cp deploy/oauth_provision /tmp/oauth_provision
```

> These files are added to a Docker image, thus this step isn't required when deploying an operator image.

Reviewer: ```suggestion
These files are added to a container image, and thus this step is not required when deploying an operator image.
```
